packages/lit-html/src/platform-support.ts

```diff
@@ -87,11 +87,11 @@ const scopeCssStore: Map<string, string[]> = new Map();
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 (globalThis as any)['litHtmlPlatformSupport'] ??= ({
-  ChildPart,
-  Template,
+  _$ChildPart: ChildPart,
+  _$Template: Template,
 }: {
-  ChildPart: PatchableChildPart;
-  Template: PatchableTemplate;
+  _$ChildPart: PatchableChildPart;
+  _$Template: PatchableTemplate;
```

Comment:
I would just change these to an argument list; doesn't seem worth burning more stable properties to make this an options bag...

Reply:
done

cc @sorvell to confirm. It means future changes just have to be adding to the list, but maybe we don't expect any.